src/workflow/unlock.h

```diff
@@ -15,25 +15,9 @@
 #ifndef _WORKFLOW_UNLOCK_H_
 #define _WORKFLOW_UNLOCK_H_
 
-#include "workflow.h"
-
 #include <compiler_util.h>
 #include <keystore.h>
 
-/**
- * Create a workflow that prompts the user for the password and unlocks the keystore, calls
- * the provided callback, and exits. The workflow will terminate if either the user enters the
- * correct password, or the device is reset after too many failed attempts. The callback will
- * receive information on whether the unlock was successful or not.
- *
- * @param[in] callback The callback that will be invoked when the workflow terminates.
- *                     It will receive a parameter that is true iff the unlock was successful,
- *                     and the user-defined parameter.
- * @param[in] callback_param The user-defined parameter that will be passed to the callback.
- * @return workflow_t object ready to be started.
- */
-workflow_t* workflow_unlock(void (*callback)(bool result, void* param), void* callback_param);
```

Comment:
What about https://github.com/digitalbitbox/bitbox02-firmware/blob/4703425b00d1244e48b1fa7fcd9fcb8033905569/test/unit-test/framework/mock_workflow.c. Is it still needed?

Reply:
Good catch. Aaand it's gone.